### PR TITLE
Modify test describes to match file paths 

### PR DESCRIPTION
**Pull request by gaurav1911**
#12

### CODE REVIEW

File: test/reporters/console/no-props.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const consoleReporter = require('../../../src/reporters/console-reporter')
 const fixture = require('../../fixtures/reporters/console/empty')
 
-describe('Console Reporter', () => {
+describe('/fixtures/reporters/console/empty', () => {
```
Comment (ovidiubute) [inline test/reporters/console/no-props.js]: This is the name of the fixture, not the file containing the code. You can just modify it to `describe('/reporters/console')`

File: test/reporters/console/empty.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const consoleReporter = require('../../../src/reporters/console-reporter')
 const fixture = require('../../fixtures/reporters/console/empty')
 
-describe('Console Reporter', () => {
+describe('/fixtures/reporters/console/empty', () => {
```
Comment (ovidiubute) [inline test/reporters/console/empty.js]: This is the name of the fixture, not the file containing the code. You can just modify it to describe('/reporters/console')

File: test/reporters/console/base.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 const consoleReporter = require('../../../src/reporters/console-reporter')
 const fixture = require('../../fixtures/reporters/console/base')
 
-describe('Console Reporter', () => {
+describe('/fixtures/reporters/console/base', () => {
```
Comment (ovidiubute) [inline test/reporters/console/base.js]: This is the name of the fixture, not the file containing the code. You can just modify it to describe('/reporters/console')

**Review by ovidiubute — CHANGES_REQUESTED** [review]

**Comment by gaurav1911** [comment]
@ovidiubute Thank you for reviewing my PR.
Updated the test describes.

**Review by ovidiubute — APPROVED** [review]

**Pull request merged**